doc/api/http.md

```diff
@@ -1185,6 +1185,13 @@ added: v0.0.2
 The `response.finished` property will be `true` if [`response.end()`][]
 has been called.
 
+### response.flushHeaders()
+<!-- YAML
+added: v1.6.0
+-->
+
+Flush the response headers.
```

Comment:
```suggestion
Flushes the response headers.
```

Reply:
It's for consistency with https://nodejs.org/api/http.html#http_request_flushheaders. Should we change both?